src/syncerd/monero_syncer.rs

```diff
@@ -271,7 +271,7 @@ async fn sweep_address(
     if balance.unlocked_balance != 0 {
         info!(
             "Sweeping address {} with unlocked balance: {}",
-            dest_address.addr(),
```

Comment:
Why not both?

Reply:
I'll add both. Just saw that there is another log error somewhere else... will fix it in this PR too

Reply:
Done in 4a638ca0bc897ece090e9d0aa48359791a920be8